webtau-core/src/main/java/org/testingisdocumenting/webtau/expectation/code/ThrowExceptionMatcher.java

```diff
@@ -103,6 +103,10 @@ private void extractExceptionDetails(Throwable t) {
             Throwable undeclaredCheckedException = t.getCause();
             thrownMessage = undeclaredCheckedException.getMessage();
             thrownClass = undeclaredCheckedException.getClass();
+        } else if (t instanceof ExceptionInInitializerError) {
```

Comment:
oO

Reply:
?

Reply:
surprised to see this, that's all. Fair change. And also signals that we may need to throw at a different time maybe to make exception more user friendly? Like maybe another check in the `persona` method?

Reply:
Yeah, I considered making the c'tor private and adding factory methods instead which would have avoided this.  I don't feel too strongly either way.

Reply:
I meant the factory method we already have and which most users will probably use. But then we need double guarding. I like your suggestion better